example/13_pool2d_fwd/pool2d_fwd_common.hpp

```diff
@@ -29,19 +31,24 @@ static void pool_host_verify(const Tensor<InDataType>& in,
                              const std::array<ck::index_t, 2>& in_left_pads,
                              const std::array<ck::index_t, 2>& /*in_right_pads*/)
 {
-    using namespace ck::host_reduce;
-
     const int32_t divider = window_spatial_lengths[0] * window_spatial_lengths[1];
 
-    const auto PreUnaryOp = PreUnaryOpFn<AccDataType, ReduceOpId>(divider);
-    const auto PosUnaryOp = PosUnaryOpFn<AccDataType, ReduceOpId>(divider);
+    using ReduceOperation = typename ck::reduce_binary_operator<AccDataType, ReduceOpId>::opType;
+    using InElementwiseOperation = typename ck::
+        reduce_unary_operator<AccDataType, ReduceOpId, true, true>::InElementwiseOperation;
+    using AccElementwiseOperation = typename ck::
+        reduce_unary_operator<AccDataType, ReduceOpId, true, true>::AccElementwiseOperation;
+
+    const InElementwiseOperation in_elementwise_op(divider);
+    const AccElementwiseOperation acc_elementwise_op(divider);
 
     if constexpr(!OutputIndex)
     {
-        auto opReduce = ReduceOpFn<AccDataType, ReduceOpId>();
+        using Accumulation =
+            ck::detail::AccumulateWithNanCheck<PropagateNan, ReduceOperation, AccDataType>;
 
         auto f_nchw = [&](auto n, auto c, auto ho, auto wo) {
-            auto accuVal = ReduceOpZeroVal<AccDataType, ReduceOpId>();
+            auto accuVal = ReduceOperation::GetReductionZeroVal();
```

Comment:
We can take the opportunity to rename `ZeroVal` to `IdentityValue`.

> IdentityValue
This value is a mathematical property of reduction type, and should be deterministic and not specified by user of reduction

https://github.com/ROCmSoftwarePlatform/composable_kernel/issues/144#issue-1178599173

Reply:
We have description of GetReductionZeroVal() in the comments of file reduction_operator.hpp about the so called `Identity value`.   But naming here considers not only algebraic theory,   we need an agreement to make a re-naming.

Reply:
I thought that discussion about renaming is more or less settled in #144. If this PR changes the interface naming anyway, we can just rename the function to match the description. A little bit of consistency helps.

Reply:
@qianfengz Please rename to ```GetIdentityValue()``` as described in #144

Reply:
Fixed by [d3813](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/262/commits/72576383569d0a115d5224b97b9274b869dd3813)